container-host-files/etc/hcf/config/scripts/patches/fix_persi_nfsdriver.sh

```diff
@@ -0,0 +1,51 @@
+set -e
```

Comment:
The file name could be more explanatory, and it could use a comment on what it does. Something like `persi_nfsdriver_user_group.sh`, and
```bash
# SUSE compatibility: this forces the `cvcap` user to have its own group, and users `usermod` instead of `adduser`
```

Reply:
I think it's better to have one file for the job, and we can comment in the file like you mentioned.